src/app/blog/(posts)/running-and-scaling-dedicated-game-servers-in-godot/page.mdx

```diff
@@ -0,0 +1,63 @@
+
+export const config = {
+  author: 'nathan-flurry',
+  date: '2024-05-09',
+  title: `Everything You Need to Know About Running & Scaling Dedicated Game Servers in Godot`,
+  description: `Learn the key aspects of running and scaling dedicated game servers for your Godot-based multiplayer game`,
+  category: 'guide',
+  keywords: [
+    'godot',
+    'multiplayer',
+    'dedicated-game-servers',
+    'server-scaling',
+    'networking-protocols',
+    'multiplayer-synchronization',
+    'testing'
+  ],
+  images: {
+    hero: {
+      image,
+      alt: 'hero'
+    }
+  }
+};
+
+## Introduction
+
+When it comes to creating a shared, multiplayer game world in Godot, setting up and managing dedicated game servers is a crucial aspect of the development process. In this blog post, we'll explore the key considerations and best practices for running and scaling dedicated game servers for your Godot-based multiplayer project.
+
+## Choosing the Right Networking Protocol
+
+When selecting a networking protocol for your Godot game, TCP is often the recommended choice. While UDP is designed for low-latency communication at the cost of reliability, it may be overkill for a simple 2D pixel game. TCP provides reliable data delivery, making it suitable for most multiplayer scenarios. If you need to support HTML5 or bypass firewalls where arbitrary TCP is restricted, WebSockets can be a great alternative.
+
+## Leveraging Godot's Multiplayer Features:
+
+Godot offers built-in multiplayer capabilities, but it's important to be aware of its limitations. Unlike mature multiplayer frameworks, Godot currently lacks client-side interpolation, which can result in jerky gameplay without additional code on your part. It's also worth noting that Godot may experience crashes when handling a large number of clients. For an in-depth exploration of Godot's multiplayer features, check out our article [link to the article].
+
+## Synchronization and State Management
+To ensure smooth synchronization between clients, a common approach is to sync the node's position on a secondary variable called server_pos. In the update loop, lerp the node's position towards the server-sent position. This technique allows for smooth movement even if updates are received a few times per second. Keep in mind that this approach may feel a bit "mushy" without extra prediction, but it's a solid starting point for most multiplayer games.
+
+## Authentication and Security
+
+When it comes to authentication and security, Godot provides authentication callbacks that you can leverage. It's recommended to set up an account system with a captcha and verify accounts via auth callbacks to mitigate botting. However, it's best to focus on shipping your game first and address these concerns once they become a problem. If you start encountering bot issues, it's a good sign that you've already accomplished the challenging task of creating a game that people are interested in playing.
```

Comment:
I think it would be better to focus on what it means when we say "Godot provides authentication callbacks". I think an extra sentence or two will help with this, but will suggest tomorrow morning.